src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -207,7 +242,8 @@ public void onTraversalStart() {
                 logger.info("Found estimated rows in streaming buffer!!! " + streamingBufferRows);
             }
 
-            LoadStatus.LoadState state = loadStatus.getSampleLoadState(Long.parseLong(sampleId));
+            LoadStatus.LoadState state = loadStatus.getSampleLoadState(sampleId);
+//            LoadStatus.LoadState state = LoadStatus.LoadState.NONE;
```

Comment:
the cruft seems to have staged a comeback

Reply:
whoops -- fixed